src/commands/lint.ts

```diff
@@ -35,6 +35,7 @@ export class LintCommand implements Command {
   // TODO(rictic): rename to 'lint' here and elsewhere, delete
```

Comment:
oh, yeah. that comment is a holdover from when the new linter was `experimental-lint`